### PR TITLE
Added Game Engines and VSCode

### PR DESCRIPTION
**Pull request by AhmadHrb**
Offers free version for starting developers.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1041,6 +1041,8 @@ Table of Contents
   * [Qoddi](https://qoddi.com) - PaaS service similar to Heroku with a developer-centric approach and all inclusive features. Free tier for static assets, staging and developer apps.
   * [FreeFlarum](https://freeflarum.com/) - Community powered free Flarum hosting for up to 250 users (donate to remove watermark from footer).
   * [fleek.co](https://fleek.co/) - Build modern sites and apps on the Open Web and its protocols seamlessly free for, unlimited websites and 50 GB/month bandwidth.
+  * [000webhost](https://000webhost.com) - Free PHP, MySQL, and cPanel Hosting.
+  * [InfinityFree](https://infinityfree.com) - Completely free website hosting with PHP, MySQL and 99% uptime.
```
Comment (ripienaar) [inline README.md:1045]: We're not adding these kinds of sites anymore, see https://github.com/ripienaar/free-for-dev/pull/2437

File: README.md [hunk 2]
```diff
@@ -1512,6 +1516,7 @@ Table of Contents
   * [RevenueCat](https://www.revenuecat.com/) — Hosted backend for in-app purchases and subscriptions (iOS and Android). Free up to $10k/mo in tracked revenue.
   * [vatlayer](https://vatlayer.com/) — Instant VAT number validation and EU VAT rates API, free 100 API requests/month
   * [Freecurrencyapi.net](https://freecurrencyapi.net/) — Free currency conversion and exchange rate data API. 10 requests/hour without an API key, 50 000 requests per month when you register for free.
+  * [Stripe](https://stripe.com) - A simple API integration used to accept payments on various platforms.
```
Comment (ripienaar) [inline README.md:1519]: Please include details of whats free

File: README.md [hunk 3]
```diff
@@ -1646,7 +1651,8 @@ Table of Contents
   * [ArtStation](https://www.artstation.com/) - MarketPlace for Free/Paid 2D, 3D assets & audios, icons, tile sets, game kits,..., Also, It can be used for showcasing your art portfolio.
   * [Rive](https://rive.app/community/) - Community assets as well as create your own game assets using its free plan. 
   * [Poly Pizza](https://poly.pizza/) - Free low poly 3D assets
-
+  * [Unity](https://unity.com) - Cross-platform game engine for real-time development
+  * [Unreal Engine](https://www.unrealengine.com/) - Advanced game engine for game creation
```
Comment (ripienaar) [inline README.md:1655]: Dont think either of these are SaaS

File: README.md [hunk 4]
```diff
@@ -1512,6 +1515,7 @@ Table of Contents
   * [RevenueCat](https://www.revenuecat.com/) — Hosted backend for in-app purchases and subscriptions (iOS and Android). Free up to $10k/mo in tracked revenue.
   * [vatlayer](https://vatlayer.com/) — Instant VAT number validation and EU VAT rates API, free 100 API requests/month
   * [Freecurrencyapi.net](https://freecurrencyapi.net/) — Free currency conversion and exchange rate data API. 10 requests/hour without an API key, 50 000 requests per month when you register for free.
+  * [Stripe](https://stripe.com) - A free payment and checkout API for cards, bank transfers with 3d secure authentication and instant payouts.
```
Comment (ripienaar) [inline README.md:1518]: I really am not sure about how free stripe is, you cannot do anything with it without paying - syaing the API is free but processing any payment incurs a cost does not make the API free

can you tell me what you found was free with it? Like something actually useful thats free

Comment (AhmadHrb) [reply README.md:1518]: I don't think there's something free it does cost a small fee per transaction. But as a service it's free to use (no monthly or annual payments needed).

Comment (ripienaar) [reply README.md:1518]: Free offering has to be useful for this list I think

**Comment by AhmadHrb** [comment]
Anything else to remove?

**Comment by ripienaar** [comment]
Thank you sorry it was such a pain!

**Comment by AhmadHrb** [comment]
> Thank you sorry it was such a pain!

No problem, happy to help!

**Pull request merged**